en/micro-integrator/docs/setup/deployment/migrating-from-ei-6.x.x.md

```diff
@@ -117,3 +117,36 @@ If you have used a custom log4j component in you you older EI version, apply the
 		```
 
 4.	Follow the instructions on [configuring log4j2](../../../administer-and-observe/logs/configuring_log4j_properties) to register the Appenders and Loggers.
+
+
+### Migrate from the Previous Releases ###
```

Comment:
This wouldn't work with EI 6.x.x right? Would it be better to add this in a separate doc?

Reply:
Shall we close this PR since it is no longer valid?